tests/console/redis.pm

```diff
@@ -20,65 +20,219 @@ use serial_terminal 'select_serial_terminal';
 use utils qw(zypper_call script_retry validate_script_output_retry);
 use registration qw(add_suseconnect_product get_addon_fullname);
 
-sub run {
-    my $self = shift;
-    select_serial_terminal;
+my $default_redis_version = "redis";
```

Comment:
Why ? there is redis and redis7, there is no default and there is no need to create variable default_redis_version